### PR TITLE
Remove locales from all the urls

### LINKED ISSUE #386: Remove locales from all the urls
We should remove locales from all the urls in project at once because:

Everytime someone makes some changes , there might be some chances that reviewer will miss the locale part.

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #386

### CODE REVIEW

File: documentation/develop.md [hunk 1]
```diff
@@ -284,7 +284,7 @@ If you have a [legacy Firefox extension](/documentation/develop/porting-a-legacy
 
 Test and [debug your code](/documentation/develop/debugging/) by [temporarily installing](/documentation/develop/temporary-installation-in-firefox/) it in Firefox.
 
-Understanding [extensions and the Add-on ID](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/WebExtensions_and_the_Add-on_ID) will help you test more effectively. You will also want to [test persistent and restart features](/documentation/develop/testing-persistent-and-restart-features/) along with [permission requests](/documentation/develop/test-permission-requests/).
+Understanding [extensions and the Add-on ID](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/WebExtensions_and_the_Add-on_ID) will help you test more effectively. You will also want to [test persistent and restart features](/documentation/develop/testing-persistent-and-restart-features/) along with [permission requests](/documentation/develop/test-permission-requests/).
```
Comment (caitmuenster) [inline documentation/develop.md]: Can you change the link for "extensions and the Add-on ID" to `/documentation/develop/extensions-and-the-add-on-id/`?

File: documentation/develop/choosing-a-firefox-version-for-extension-development.md [hunk 2]
```diff
@@ -102,19 +102,19 @@ Suggested development use: You may want to install and test a signed copy of you
...
+You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties (generally fewer changes than you need to achieve the same tasks in Nightly). This enables you to test features such as restart behavior, permission prompts, and upgrades.
```
Comment (caitmuenster) [inline documentation/develop/choosing-a-firefox-version-for-extension-development.md]: Can you change the link for "web-ext" to `/documentation/develop/getting-started-with-web-ext/`?

File: documentation/develop/choosing-a-firefox-version-for-extension-development.md [hunk 3]
```diff
@@ -102,19 +102,19 @@ Suggested development use: You may want to install and test a signed copy of you
...
 
-You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/en-US/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/en-US/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties. This enables you to test features such as restart behavior, permission prompts, and upgrades.
+You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties. This enables you to test features such as restart behavior, permission prompts, and upgrades.
```
Comment (caitmuenster) [inline documentation/develop/choosing-a-firefox-version-for-extension-development.md]: Please change web-ext link to `/documentation/develop/getting-started-with-web-ext/`.

File: documentation/develop/choosing-a-firefox-version-for-extension-development.md [hunk 4]
```diff
@@ -126,13 +126,13 @@ Suggested development use: Nightly provides a preview of future Firefox features
...
+You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties (this feature is provided so enterprises can install extensions they want to keep private and don’t want to submit to add-ons.mozilla.org for signing).
```
Comment (caitmuenster) [inline documentation/develop/choosing-a-firefox-version-for-extension-development.md]: Please change web-ext link to `/documentation/develop/getting-started-with-web-ext/`.

File: documentation/enterprise.md [hunk 5]
```diff
@@ -98,7 +98,7 @@ Developing extensions for your enterprise is straightforward – you can follow
 
 Extensions for enterprise deployments of Firefox Extended Support Release (ESR) should be distributed outside of addons.org.
 
-Once your extension is developed and tested, you can use one of the [enterprise distribution](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Alternative_distribution_options/Add-ons_in_the_enterprise) methods to install it in Firefox.
+Once your extension is developed and tested, you can use one of the [enterprise distribution](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Alternative_distribution_options/Add-ons_in_the_enterprise) methods to install it in Firefox.
```
Comment (caitmuenster) [inline documentation/enterprise.md]: Can you change this link to `/documentation/enterprise/enterprise-distribution/`?

File: documentation/publish.md [hunk 6]
```diff
@@ -63,7 +63,7 @@ Once your extension is signed, it is subject to review by Mozilla at any time. T
 
 There are four ways you can distribute your extension. They are:
 
-- Use [addons.mozilla.org](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on) (most popular, broadest audience)
+- Use [addons.mozilla.org](https://developer.mozilla.org/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on) (most popular, broadest audience)
```
Comment (caitmuenster) [inline documentation/publish.md]: Please change link to `/documentation/publish/submitting-an-add-on/`.

File: documentation/publish/signing-and-distribution-overview.md [hunk 7]
```diff
@@ -68,7 +68,7 @@ Unsigned extensions can be installed in [Developer Edition](https://www.mozilla.
 
 Mozilla signs add-ons through the [AMO](https://addons.mozilla.org) website and provides three methods for submitting your add-on for signing:
 
-- upload your add-on through the [Developer Hub on AMO](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on).
+- upload your add-on through the [Developer Hub on AMO](https://developer.mozilla.org/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on).
```
Comment (caitmuenster) [inline documentation/publish/signing-and-distribution-overview.md]: Please change link to `/documentation/publish/submitting-an-add-on/`.

File: documentation/publish.md (hunk 6 above)
Comment (championshuttler) [reply documentation/publish.md]: Oops just noticed, is text quite clear? 

Use addons.mozilla.org (most popular, broadest audience) and we are linking amo to submitting-an-add-on.

Just cross checking

File: documentation/publish/signing-and-distribution-overview.md (hunk 7 above)
Comment (championshuttler) [reply documentation/publish/signing-and-distribution-overview.md]: Same here

Comment (caitmuenster) [reply documentation/publish/signing-and-distribution-overview.md]: Good question -- we can be more specific and link to `/documentation/publish/submitting-an-add-on/#listing-on-amo`.

File: documentation/publish.md (hunk 6 above)
Comment (championshuttler) [reply documentation/publish.md]: The last one for you @caitmuenster ;)

**Pull request merged**